decisions/Gondor.txt

```diff
@@ -1789,6 +1789,33 @@ political_decisions = {
 		}
 	}
 	
+	claim_harnendor = {
+		picture = "harnendor_map"
+		potential = {
+			tag = GON
+			HRD = { all_core = { owned_by = GON } }
+			NOT = {
+				has_global_flag = harnendor_claimed_GON
+			}
+		}
+		
+		allow = {
+			war = no
+		}
+		
+		effect = {
+			prestige = 10
+			badboy = 3
+			HRN = {
+				all_core = {
+					add_core = GON
+					add_core = HRD
+				}
+			}
```

Comment:
In reference to our Discord conversation earlier, HRN cores != the Harnendor that Gondor once owned. 
 - HRN = the Haradrim tribe (that I happened to give a matching name to) 
 - HAR_286 = the region that Gondor calls Harnendor

HAR_286 = { all_core = { add_core = GON } }

We should also probably accommodate for the possibility of Harondor cores being removed (which I think does happen if the player follows flavor)
```
# Adds Harondor cores to Harnendor (HAR_286) if Harondor cores haven't been removed
random_owned = {
    limit = { province_id = 249 }
    owner = { HAR_286 = { all_core = { add_core = HRD} } }
}
```

I believe that should do the trick